### DISCUSSION TITLE
Meta Data

### DISCUSSION THREAD

**Discussion by PH5HP** [rejetto/hfs#465] [General]
Hi,

I am trying to add my own meta data, but I can not put it in the <head> section.The nearest section to the <head> I can put it in is the 'Top of the HTML Body'. Would you consider a new section in Custom HTML to allow meta data in the <head> section of the HTML layout?

Kind regards,
Henk - fileserver.ph5hp.nl

**Comment by rejetto** [comment]
please give an example, i'm not understanding

**Reply by PH5HP** [reply]
Hi,

Meta is used by search engines to list the HFS server in the search results. HFS it self generates also some meta data. See the screenshot attached. In the HEAD section you see some lines with META at the beginning.

I want to add my own meta information in that section, but I can not do this at the moment. The closest to the HEAD section is the 'Top of the HTML Body' section in 'Custom HTML'.

You can see my own meta data also in the screenshot. All lines beginning with 'meta'. These lines belong in the HEAD section like the lines generated by HFS.

Kind regards,
Henk - fileserver.ph5hp.nl
![image](https://github.com/rejetto/hfs/assets/140949730/34abc4a8-c789-46cc-a928-c6d3db5878fa)

**Comment by rejetto** [comment]
ok, html's meta data.
i thought you were talking about files' meta data. 😬
right, i guess i need to add a <head> section.
should come soon.
keep you posted.

**Reply by PH5HP** [reply]
Hi,

Thank you !

Kind regards,
Henk - fileserver.ph5hp.nl

**Comment by rejetto** [comment]
https://github.com/rejetto/hfs/releases/tag/v0.52.0-alpha2

**Reply by PH5HP** [reply]
Hi,

Works great. Thank you.

Kind regards,
Henk - fileserver.ph5hp.nl
